Bluetooth: Mesh: Point to point and Mesh DFU (#14833)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/dfu/target/README.rst

* `nRF Device Manager mobile app for Android`_
* `nRF Device Manager mobile app for iOS`_

Point-to-point DFU requirements
*******************************

The configuration overlay :file:`overlay-ptp_dfu.conf` enables the :ref:`dfu_over_ble` feature.

> This is added as a backup option for the DFU process in case the mesh network is not available.

Reviewer: ```suggestion
This feature can be used as a backup option for the DFU process if the mesh network is not available.
```

> This feature can be used as a backup option for the DFU process if the mesh network is not available.

Reviewer 2: Could we clarify usage of them a bit? I mean impossibility to use in parallel. Or for example that, one procedure will not pick up dfu process of another if it was interrupted\suspended.

Author: @mia-ko : I changed the text here, could you check what I wrote?

* `nRF Device Manager mobile app for Android`_
* `nRF Device Manager mobile app for iOS`_

Point-to-point DFU requirements
*******************************

The configuration overlay :file:`overlay-ptp_dfu.conf` enables the :ref:`dfu_over_ble` feature.

This feature can be used together with Bluetooth Mesh DFU.

> If the Bluetooth Mesh DFU procedure is suspended, failing or if the Bluetooth Mesh network is not available, the point-to-point DFU feature can be used as a backup option for the DFU process.

Reviewer: ```suggestion
If the Bluetooth Mesh DFU procedure is suspended, failing, or if the Bluetooth Mesh network is not available, the point-to-point DFU feature can be used as a backup option for the DFU process.
```
